src/PharChecker.php

```diff
@@ -22,6 +22,15 @@
      */
     public function checkFileValidity($filename)
     {
+        if (!is_string($filename)) {
```

Comment:
we don't check primitive types described in docblocks anywhere, and they will be strongly typehinted on 7+

Reply:
lets drop this when we are on PHP7 than

Reply:
but we don't do any of those checks, anywhere...
either everywhere or nowhere, staying in the middle is weird

Reply:
removed